plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/util/CloudSdkProcessWrapperTest.java

```diff
@@ -39,34 +38,25 @@ public class CloudSdkProcessWrapperTest {
 
   private final CloudSdkProcessWrapper wrapper = new CloudSdkProcessWrapper();
 
+  @Before
+  public void setUp() {
+    TestAccountProvider.setAsDefaultProvider(State.NOT_LOGGED_IN);
```

Comment:
As long as a test suite does not instantiate a client (e.g. `apiFactory.newXXXClient()`), this account provider must be setup to prevent actual calls or credential checks